applications/OptimizationApplication/python_scripts/algorithms/standardized_NLOPT_constraint.py

```diff
@@ -22,10 +24,16 @@ class StandardizedNLOPTConstraint(ResponseRoutine):
 
     """
     def __init__(self, parameters: Kratos.Parameters, master_control: MasterControl, optimization_problem: OptimizationProblem, required_buffer_size: int = 2):
+        # backward compatibility
+        if parameters.Has("response_name"):
+            IssueDeprecationWarning(self.__class__.__name__, "\"response_name\" is deprecated. Please use \"response_expression\".")
+            parameters.AddString("response_expression", parameters["response_name"].GetString())
+            parameters.RemoveValue("response_name")
+
         default_parameters = Kratos.Parameters("""{
-            "response_name": "",
-            "type"                     : "",
-            "ref_value"             : "initial_value"
+            "response_expression": "",
```

Comment:
I don't understand why do you need this changes. we can discuss later

Reply:
Now, it is not only the name of the response which is accepted, it accepts a resposne expression. Thats the reason why i changed it. This is backward compatible. (I added a deprecation warning)